renderite/serialization.zig

```diff
@@ -114,7 +150,7 @@ pub const IpcDeserializer = struct {
 
     pub fn readValueList(self: IpcDeserializer, comptime T: type, gpa: std.mem.Allocator) ![]T {
         const len = try self.reader.takeInt(i32, endian);
-        if (len == 0 or len == -1) // TODO: handle -1 meaning null
+        if (len == 0)
```

Comment:
Why were the -1 checks removed?

Reply:
I was reading invalid data that just happened to be `-1`. I read similar code somewhere and got it wrong. This doesn't actually happen in the memory packer from what I can tell.